server.py

```
@@ -1183,6 +1396,27 @@ def get_submission_by_url(url: str) -> Dict[str, Any]:
+                submission.comments.replace_more(limit=None)
+
+                top_level_comments = [
+                    c
+                    for c in submission.comments
+                    if isinstance(c, praw.models.Comment)
+                ]
+
+                submission_data["comments"] = [
+                    _serialize_comment_tree(c) for c in top_level_comments
+                ]
+            except Exception as comments_error:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)